modules/core/test/v2/unit/coins/utxo/fixtures/btc/transactions-p2tr.json

```diff
@@ -0,0 +1,162 @@
+{
+  "prebuild": {
+    "id": "9d9d502ef0281ea6d9ff68d75ddae5fe5dd68b97b68080174e1911830cac58cd",
+    "hex": "0100000002ea49605353702c351de441c86930341ac3f0c4f12143ab33cccf67fadd95f29c0100000000ffffffffe14a1d03237d57b49989fc9afdb10ace84428e0d60f1f5506c6cbef89954536f0100000000ffffffff0118beeb0b0000000017a9141e57a925dd863a86af341037e700862bf66bf7b68700000000",
+    "ins": [
+      {
+        "txid": "9cf295ddfa67cfcc33ab4321f1c4f0c31a343069c841e41d352c7053536049ea",
+        "index": 1,
+        "script": "",
+        "witness": []
+      },
+      {
+        "txid": "6f535499f8be6c6c50f5f1600d8e4284ce0ab1fd9afc8999b4577d23031d4ae1",
+        "index": 1,
+        "script": "",
+        "witness": []
+      }
+    ],
+    "outs": [
+      {
+        "script": "a9141e57a925dd863a86af341037e700862bf66bf7b687",
+        "value": 199999000
+      }
+    ]
+  },
+  "halfSignedUserBackup": {
+    "id": "9d9d502ef0281ea6d9ff68d75ddae5fe5dd68b97b68080174e1911830cac58cd",
+    "hex": "01000000000102ea49605353702c351de441c86930341ac3f0c4f12143ab33cccf67fadd95f29c0100000000ffffffffe14a1d03237d57b49989fc9afdb10ace84428e0d60f1f5506c6cbef89954536f0100000000ffffffff0118beeb0b0000000017a9141e57a925dd863a86af341037e700862bf66bf7b687040041a489417f71cae151810d2d55b1d97d6949ad0f3a57f87420983a3ee7c69f35f57ad103cb7c6a0b1fad8abd86e84cf619270ff998c499d2d881982137c3d3041c0144201ce1b0b4d5d856368637b75808b138f042b335d23bb46d297f19b09000a8612ead20d4c149c56e614423aa7c55c91d71b9b30dd8e8af29b9e04fc003dcc8ed3c3bbfac61c0312363917d7b7b60c312a0a2fa761f885a76746769436ccd1ff042c9e9dc81495d503a0d622e4c5a538fc4061d41a4d76943880028022adf565a2b5e7df0d083d0cb8bfdb6c6e7213b38bca5978bd646de4a79848028a610ef92e4d7a302603d04004155c7eb58ebbd154b897a81ccd1b07f04f226e540a5a21158588cbc176224fc0cc33cf36a7b197072982b707cd662bcebd5cf4a7d974001846a933c2ab06bcbdc014420be38da53fc0c51074bc239855667a8a457000ee2173ef2041927d0eeb016787aad201fd7e44980cbfd63b30d1ce141333242cf4f0080447e4db628f38e4bc5c09106ac61c1d8415bc416f85fde3688bea33bfa82a24b8342ff1c837818093a31df478d63471f6a8139ffdd5120ed77cd33a1a3d478c294f7114c8c9343217f58f01cabb15532b0389bb3246784a2631d3904f2bddf500dd2d40b98cb2c65e70ecf260a9dcf00000000",
+    "ins": [
+      {
+        "txid": "9cf295ddfa67cfcc33ab4321f1c4f0c31a343069c841e41d352c7053536049ea",
+        "index": 1,
+        "script": "",
+        "witness": [
+          "",
+          "a489417f71cae151810d2d55b1d97d6949ad0f3a57f87420983a3ee7c69f35f57ad103cb7c6a0b1fad8abd86e84cf619270ff998c499d2d881982137c3d3041c01",
```

Comment:
I see we're using SIGHASH_ALL instead of SIGHASH_DEFAULT here. Might want to test both, and prefer SIGHASH_DEFAULT to save the bytes.

Reply:
Actually, looking at the code, I don't see where the SIGHASH_ALL is coming from. Looks like we're passing `undefined` for `hashType` which should give `SIGHASH_DEFAULT`. Hmm...

Reply:
Found it -- not a critical bug, but it's coming from https://github.com/BitGo/BitGoJS/blob/master/modules/core/src/v2/coins/abstractUtxoCoin.ts#L1234

Reply:
it's not a parameter in `wallet.signTransaction` or `coin.signTransaction` though, and I don't think it is worthwhile adding it

BitGoJS should pick the appropriate default

Reply:
Looks like the default is only per network not per input type. We can definitely launch as is, but maybe make a ticket to fix the default for p2tr inputs.

Reply:
(or we could change the default for bitcoin to `undefined` and see what that does, since bitcoinjs-lib and HSM will both handle that fine afaik)

Reply:
https://github.com/BitGo/BitGoJS/commit/d2a8c7b1aadd77e3da08981fa462d8cf403c661b

Reply:
Nice!